src/views/About.vue

```diff
@@ -1,28 +1,42 @@
 <template>
   <div class="about">
-    <Header />
```

Comment:
Did you mean to remove the header?

Reply:
Yeah, it was taking up a lot of space on a pretty minimal page - I figure it won't hurt much since this is only a two page app